common/plugins/eu.esdihumboldt.hale.common.align/plugin.xml

```diff
@@ -445,6 +445,31 @@
             <parameterBinding></parameterBinding>
          </functionParameter>
       </propertyFunction>
+      <propertyFunction
```

Comment:
Please move the function definition to the bundle you implemented the function in - only the core functions are defined here.